shared/util/container.js

```diff
@@ -11,7 +13,10 @@ const createImmutableEqualSelector = createSelectorCreator(defaultMemoize, I.is)
 
 const NullComponent = () => null
 
-export {connect} from 'react-redux'
+// flowlint deprecated-type:off
+const storybookableConnect = (name: string, mapStateToProps: *, mapDispatchToProps: *, mergeProps: *) =>
```

Comment:
:sheep: It's a pretty generic thing this does (also names the component in the virtual dom), maybe `namedConnect` instead?